composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "chubbyphp/chubbyphp-lazy-command",
```

Comment:
I suggest, since the package is archived, to get rid of this dependency and adjust the project (feed-importer) by not using the LazyCommand (see: [mp-core example](https://github.com/jobcloud/marketplace-api-core/blob/master/app/ServiceProvider/CommandServiceProvider.php)). 
Imo, the overhead generated by not using the LazyCommand feature when executing `/bin/console` is so small that it's not worth the effort of keeping and maintaining a fork of this library.

Reply:
I agree with you, given that we have 2 PRs from here where it makes sense to drop them from the project and just integrate/use functionality from elsewhere, should I do it in this set of PRs or do this as a separate change ?

Reply:
I would do it in a dedicated one.

Reply:
Pleases create a ticket for this removal @gideonb234. Thanks.

Reply:
Pleases create a ticket for this removal @gideonb234. Thanks.

Reply:
https://jobcloud.atlassian.net/browse/IFP-430 ticket created